src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -470,7 +470,7 @@ public function getEvent()
     public function hasPermission()
     {
         if (!$this->parameters->has('permission')) {
-            return true;
+            return false;
         }
 
         return false !== $this->parameters->get('permission');
```

Comment:
The method `hasPermission` is confusing and actually it checks if permission is disabled. Maybe a better name for the method would be `isPermissionCheckDisabled`? /cc @pjedrzejewski